project/AppDependencies.scala

```diff
@@ -22,19 +22,19 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-backend-play-27" % "2.20.0",
-    "uk.gov.hmrc"   %% "domain"            % "5.9.0-play-27",
+    "uk.gov.hmrc" %% "bootstrap-backend-play-27" % "3.2.0",
+    "uk.gov.hmrc"   %% "domain"            % "5.10.0-play-27",
     //"uk.gov.hmrc"   %% "play-hmrc-api"     % "4.1.0-play-26",
-    "org.typelevel" %% "cats-core"         % "2.2.0",
+    "org.typelevel" %% "cats-core"         % "2.3.0",
     compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.1" cross CrossVersion.full),
     "com.github.ghik" % "silencer-lib" % "1.7.1" % Provided cross CrossVersion.full,
     "com.chuusai"   %% "shapeless"         % "2.4.0-M1"
   )
 
   def test(scope: String = "test, it"): Seq[sbt.ModuleID] = Seq(
-    "org.scalatest"          %% "scalatest"          % "3.2.2"             % scope,
+    "org.scalatest"          %% "scalatest"          % "3.2.3"             % scope,
```

Comment:
scalatest should be bumped to 3.3.0-SNAP2. Also jettyFromWiremockVersion should be bumped to 9.4.35.v20201120. The one we are using currently is 2018 version.

Reply:
we don't typically upgrade to non-main releases, like snapshots etc. I'll keep scalatest at 3.2.3